packages/mobile/src/screens/track-screen/DownloadRow.tsx

```diff
@@ -0,0 +1,85 @@
+import { useCallback } from 'react'
+
+import { cacheTracksSelectors } from '@audius/common'
+import type { DownloadQuality, CommonState, ID } from '@audius/common'
+import { View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import { IconReceive } from '@audius/harmony-native'
+import { Text } from 'app/components/core'
+import { PlainButton } from 'app/harmony-native/components/Button/PlainButton/PlainButton'
+import { makeStyles } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+import { useThemeColors } from 'app/utils/theme'
+
+const { getTrack } = cacheTracksSelectors
+
+const messages = {
+  fullTrack: 'Full Track'
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  root: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    borderTopWidth: 1,
+    borderTopColor: palette.borderDefault,
+    padding: spacing(4)
+  },
+  content: {
+    flexDirection: 'row',
+    gap: spacing(6),
+    alignItems: 'center',
+    justifyContent: 'space-between'
+  },
+  titleContainer: {
+    gap: spacing(1)
+  }
+}))
+
+type DownloadRowProps = {
+  trackId: ID
+  quality: DownloadQuality
+  // onDownload: (trackId: ID, category?: string, parentTrackId?: ID) => void
```

Comment:
Seeing this commented code in a few places in this file. Meant to be uncommented?